test/kubemark/resources/start-kubemark-master.sh

```diff
@@ -241,6 +241,52 @@ function prepare-log-file {
 	chown root:root $1
 }
 
+# A helper function for removing salt configuration and comments from a file.
+# This is mainly for preparing a manifest file.
+#
+# $1: Full path of the file to manipulate
+function remove-salt-config-comments {
+	# Remove salt configuration.
+	sed -i "/^[ |\t]*{[#|%]/d" $1
+	# Remove comments.
+	sed -i "/^[ |\t]*#/d" $1
+}
+
+# Prepare the abac policy file from the salt template.
+function setup-abac-authz-policy-file {
```

Comment:
I might be missing some context here, but to be honest, I don't think need anything particular from ABAC. The only reason why we are adding RBAC is that we are using it in real clusters and it may affect performance (though I guess the impact shouldn't be significant). Since kubemark is supposed to mimic real cluster, we also want to have it here.
If you think we should remove abac and leave rbac only, that definitely sounds good to me.

@shyamjvs - please fix. Also kubemark tests didn't pass, which is most probably related to your changes.